### ISSUE TITLE
Wrong password password is entered during Auto-Type

### ISSUE THREAD

**Issue by M86xKC** [keepassxreboot/keepassxc#10207]
## Overview
[TIP]:  # ( DO NOT include screenshots of your actual database! )
[NOTE]: # ( Give a BRIEF summary about your problem )

I generated a random password using KeePassKC for a site, but when trying to trigger auto-type for this site. It enters some characters then removes some of them then and then continues with the last part. I believe it is an edge case where the randomly generated password is somehow causing the auto-type to remove some characters.

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. Create a database
2. Create an entry 
3. Set the password to: 
```
d3V_{:""@AbGC~{hpLlu4,[8d8F`wjOt
```
4. Trigger auto-type

## Expected Behavior
it should auto type the password correctly

## Actual Behavior
It enters some characters then removes some of them then and then continues with the last part.

## Context
I'm using the default auto-type sequence in case that changes anything.

```
{USERNAME}{TAB}{PASSWORD}{ENTER}
```

When I copy the password and paste it in the password field I have no issue.

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - Version 2.7.6
Revision: dd21def

[NOTE]: # ( Pick choices based on your environment )
Operating System: Linux
Desktop Env: KDE
Windowing System: X11

**Comment by droidmonkey** [comment]
It is very likely that the website itself is doing the deleting. What site is it? Can you manually type in the password? Are you using an English keyboard layout?

**Comment by M86xKC** [comment]
> It is very likely that the website itself is doing the deleting.

No I tested that on a simple html page

> What site is it?

[github.com](https://github.com/login)

> Can you manually type in the password?

Surprisingly it does not work when I type it in manually but works with copy-paste.

> Are you using an English keyboard layout?

Yes, I'm using an English keyboard layout

I investigated more here: https://m86xkc.github.io/keepassxc-test/

Here is the difference between the password in KeePassXC and the final output in the input field. from looking at the diff it seems it is `:""` is causing the issue.

```diff
+d3V_{:""@AbGC~{hpLlu4,[8d8F`wjOt
-d3V_{@AbG＂C~{hpLlu4,[8d8F`wjOt
```
